doc/src/modules/holonomic/represent.rst

```diff
@@ -0,0 +1,40 @@
+Represnting a Holonomic Function in SymPy
+=========================================
+
+.. module:: sympy.holonomic.holonomic
+
+Class :class:`DifferentialOperator` is used to represent the annihilator
+but we create differential operators easily using the function
+:func:`DifferentialOperators`. Class :class:`HolonomicFunction` represents a holonomic function.
+
+Let's explain this with an example:
+
+Take :math:`sin(x)` for instance, the differential equation satisfied by it
+is :math:`y''(x) + y(x) = 0`. By definition we conclude it is a holonomic
+function. The general solution of this ODE is
+:math:`C_{1}sin(x) + C_{2}cos(x)` but to get :math:`sin(x)` we need to
+provide initial conditions i.e. :math:`y(0) = 0, y'(0) = 1`.
+
+To represent the same in this module one needs to provide the differential
+equation in the form of annihilator. Basically a differential operator is an
+operator on functions that differentiates them. So :math:`D^{n}.y(x) = y^{n}(x)`
```

Comment:
Derivatives are usually indicated by placing the order in parentheses: `y^{(n)}`